lib/dcsotiesplunk/config.py

```diff
@@ -1,36 +1,35 @@
-# Copyright (c) 2020, DCSO GmbH
-
+# Copyright (c) 2020, 2023, DCSO GmbH
 from configparser import ConfigParser
 from typing import NoReturn, Optional, Any, Union
 
 from dcsotie.errors import TIEConfigError
 from dcsotie.utils import Range, SeverityRange, ConfidenceRange
 
 CONFIG_SCHEMA = {
-    'filter': {
-        'ip_confidence': (ConfidenceRange, 80),
-        'ip_severity': (SeverityRange, 1),
-        'dom_confidence': (ConfidenceRange, 80),
-        'dom_severity': (SeverityRange, 1),
-        'url_confidence': (ConfidenceRange, 80),
-        'url_severity': (SeverityRange, 1),
-        'email_confidence': (ConfidenceRange, 80),
-        'email_severity': (SeverityRange, 1),
-        'confidence': (ConfidenceRange, 80),
-        'severity': (SeverityRange, 2),
+    "filter": {
+        "ip_confidence": (ConfidenceRange, 80),
+        "ip_severity": (SeverityRange, 1),
+        "dom_confidence": (ConfidenceRange, 80),
+        "dom_severity": (SeverityRange, 1),
+        "url_confidence": (ConfidenceRange, 80),
+        "url_severity": (SeverityRange, 1),
+        "email_confidence": (ConfidenceRange, 80),
+        "email_severity": (SeverityRange, 1),
+        "confidence": (ConfidenceRange, 80),
+        "severity": (SeverityRange, 2),
+    },
+    "tie": {
+        "client_id": (str, ""),
+        "client_secret": (str, ""),
+        "feed_api": (str, "https://api.dcso.de/tie/v1/iocs"),
```

Comment:
is this feed api string still the same?

Reply:
not sure what you mean... this is the new api string

Reply:
perhaps: tie/v1/iocs -> is still the Tie-v1 used?

Reply:
This uses the new TIE software (note that it's `api.dcso.de`, not `tie.dcso.de`) but its  "v1" API flavour, not the TAXII ("v2") one. TAXII is currently not in the state to be used productively for a case like this.
The URL is correct.